### ISSUE TITLE
Portable version

### ISSUE THREAD

**Issue by gczark** [minbrowser/min#1102]

**Comment by Syndamia** [comment]
@gczark Please provide a more detailed description of your issue.

By portable version do you mean a version that you can install on a **USB stick** and just plug in a computer and use `Min` straight away from the stick?

**Comment by gczark** [comment]
Yup! More like compiled as a zip, and include all the necessary files to run `Min`. Rather than releasing an one-click installation.

**Comment by jiangzhenjerry** [comment]
I would like to second this request, particularly related to the hope that it can contain its settings outside of the system drive. The key is to enable a parameter such as --user-data-dir=XXX that some other Electron-based apps and Chrome itself are using. Right now I just create a symbolic link to point %AppData%\Min to a folder I prefer, as an alternative.

**Comment by deprale** [comment]
> I would like to second this request, particularly related to the hope that it can contain its settings outside of the system drive. The key is to enable a parameter such as --user-data-dir=XXX that some other Electron-based apps and Chrome itself are using. Right now I just create a symbolic link to point %AppData%\Min to a folder I prefer, as an alternative.

Is this a thing now or nobody bothered ?

**Comment by zcorn2017** [comment]
Any updates?

**Comment by skbnoqtrjf** [comment]
> By portable version do you mean a version that you can install on a **USB stick** and just plug in a computer and use `Min` straight away from the stick?

yes, I want this version, how to install like this?

**Comment by TetraTheta** [comment]
Here is a suggestion about 'portable':

Visual Studio Code supports ['portable mode'](https://code.visualstudio.com/docs/editor/portable), by creating `data` directory alongside with `Code.exe`. `data` directory will also override `--user-data-dir` and `--extensions-dir` of VS Code too.

Since both VS Code and Min uses Electron, I think it would be easy to implement.
